packages/flutter/test/material/expansion_tile_test.dart

```diff
@@ -329,10 +329,11 @@ void main() {
     const Key child0Key = Key('child0');
     const Key child1Key = Key('child1');
 
-    await tester.pumpWidget(MaterialApp(
-      home: Material(
-        child: Center(
-          child: ExpansionTile(
+    // TODO(goderbauer): Reevaluate the following ignores when https://github.com/dart-lang/sdk/issues/51800 is fixed.
+    await tester.pumpWidget(MaterialApp( // ignore: prefer_const_constructors
+      home: Material( // ignore: prefer_const_constructors
```

Comment:
Are these going to show up in any customer code that uses prefer_const_constructors?  This lint is turned on as part of `flutter_lints`, so it might affect a lot of people if so.

Reply:
Yeah, they will see this once they bump the sdk constraints to 3.0 in their pubspecs. I am pushing hard on https://github.com/dart-lang/sdk/issues/51800 to get this fixed before Dart 3 is released to stable.

Reply:
Yeah, that seems important.